lib/orders.dart

```diff
@@ -107,15 +97,11 @@ class _OrderScreenState extends State<OrderScreen> {
               subtitle: Text('${describeEnum(_order.status).toUpperCase()}')),
           _order.status == BeOrderStatus.created
               ? ListTile(
-                  title:
-                      raisedButton(onPressed: _accept, child: Text('Accept')))
-              : SizedBox(),
-          _order.status != BeOrderStatus.expired &&
-                  _order.status != BeOrderStatus.failed &&
-                  _order.status != BeOrderStatus.completed
-              ? ListTile(
-                  title:
-                      raisedButton(onPressed: _update, child: Text('Update')))
+                  title: BronzeRoundedButton(_accept, ZapOnPrimary, ZapPrimary,
```

Comment:
put this in a Row widget (eg like in `class _CryptoWithdrawalDetailScreenState`) so that it repspects ButtonWidth